crates/runtime/src/logic.rs

```diff
@@ -81,6 +81,11 @@ impl<'a> VMLogic<'a> {
         }
         .build()
     }
+
+    pub fn get_executor_identity(&mut self, register_id: u64) -> Result<()> {
+        self.registers
+            .set(self.limits, register_id, self.context.executor_public_key)
+    }
```

Comment:
Again, as [mentioned](https://github.com/calimero-network/core/pull/536#discussion_r1694854727)

![CleanShot 2024-08-01 at 18 25 51@2x](https://github.com/user-attachments/assets/c8b1d6a1-b22e-4e74-9f1c-6e9c5c0f8c0e)

Like the other host functions, it should be part of `VMHostFunctions` and exported via `logic/imports.rs`

To match the guest's host function expectations.

Otherwise, in this form, it's simply unused.